src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/JiraCoreScenario.kt

```diff
@@ -81,7 +92,9 @@ class JiraCoreScenario constructor() : Scenario {
             viewDashboard to 15, // note that we may end up generating more views if we need to display top nav
             editIssue to 5,
             addComment to 2,
-            browseProjects to 5
+            browseProjects to 5,
+            viewFocusedComment to 5,
+            viewHistoryTabAction to 5
```

Comment:
As menitoned in our offline discussion I'm not sure if we should extend the existing scenario. Does it affect our API somehow? If so should it be noted in CHANGELOG.md?

Reply:
At the same time we need to ensure that the action is part of some test in this repo.

Reply:
Well, I'm also not sure if I should add new actions to existing scenario or create a new one. I don't think this may break something, but at the same time I'm not sure how (and if at all) this library is used besides JPT. Even if I go with creating a new scenario, there's still a change to the existing one - View Issue action got additional memory for comments. 
Maybe other reviewers have some recommendation? cc/ @lipinskipawel @dagguh @wyrzyk

Reply:
Extending the `JiraCoreScenario` coverage is valuable and compatible. Switching those tabs **is** a part of a Jira Core user scenario. I understand we might be afraid of introducing flakiness or bugs, but:
* we should quickly identify and fix those
* the consumer can roll back to old lib version

Reply:
What was the existing behavioral contract?
> Javadoc comment: `Provides Jira Core specific Scenario`
> class name `JiraCoreScenario`

Viewing focused comment and viewing history tab still fit. We never promised *not* to do more actions.